test/fungus_toast/games/games_test.exs

```diff
@@ -3,8 +3,9 @@ defmodule FungusToast.GamesTest do
 
   alias FungusToast.Accounts
   alias FungusToast.Games
+  alias FungusToast.Players
 
-  def user_fixture(attrs \\ %{}) do
+  defp user_fixture(attrs \\ %{}) do
     {:ok, user} =
       attrs
       |> Enum.into(%{user_name: "testUser"})
```

Comment:
Is this being used? Just from the REPL, `Enum.into` doesn't return `{:ok, ...}` or `{:error, ...}`

Reply:
Accidentally resolved this. The Enum.into feeds into an Accounts.create that doesn't seem to be in the diff

Reply:
Ahh - 🤦‍♂️ good call. I had to view file to see it.